lib/matplotlib/animation.py

```diff
@@ -1723,7 +1722,25 @@ def new_saved_frame_seq(self):
             self._old_saved_seq = list(self._save_seq)
             return iter(self._old_saved_seq)
         else:
-            return itertools.islice(self.new_frame_seq(), self.save_count)
+            it = itertools.islice(self.new_frame_seq(), self.save_count)
+
+            def gen():
+                try:
+                    for _ in range(100):
+                        yield next(it)
+                except StopIteration:
+                    return
+                cbook.warn_deprecated(
+                    "2.2", "FuncAnimation.save has truncated your animation "
+                    "to 100 frames.  In the future, no such truncation will "
+                    "occur; please pass 'save_count' accordingly.")
+                try:
```

Comment:
Sorry to be a pain, but why is this second block needed?  Are you going to keep returning frames even though the saving is not saving any more?

Reply:
because I'm an idiot...